### PR TITLE
Updates the Prerequisite Snippet

### PR DESCRIPTION
**Pull request by JeffNeff**
Part of Issue #4191 

## Proposed Changes 

- Updates the suggested minimum version for K8s in the `Prerequisites` snippet.

### CODE REVIEW

**Comment by JeffNeff** [comment]
how does this look @csantanapr ? Do you think it's okay to link the Versioning Principales page here as opposed to just updating the version number?

File: docs/snippets/prerequisites.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Before installing Knative, you must meet the following prerequisites:
 
     - If you have only one node in your cluster, you need at least 6&nbsp;CPUs, 6&nbsp;GB of memory, and 30&nbsp;GB of disk storage.
     - If you have multiple nodes in your cluster, for each node you need at least 2&nbsp;CPUs, 4&nbsp;GB of memory, and 20&nbsp;GB of disk storage.
-- You have a cluster that uses Kubernetes v1.18 or newer.
+- You have a cluster that uses a supported Kubernetes version. Refer to the [Release Versioning Principles](https://github.com/knative/community/blob/main/mechanics/RELEASE-VERSIONING-PRINCIPLES.md){_blank} page for more information.
```
Comment (antoineco) [inline docs/snippets/prerequisites.md:14]: What does "_blank" mean here? First time I see it.

Comment (JeffNeff) [reply docs/snippets/prerequisites.md]: I'm testing out GitHub copilot and.. well I might have to turn it off lol. Sometimes it slips in odd things without me noticing. It's been removed [5f8771d](https://github.com/knative/docs/pull/4196/commits/5f8771d0b8a9e0c54d74bb4686714d7d704fa774)

**Review by antoineco — APPROVED** [review]

**Comment by sebgoa** [comment]
this was fixed in https://github.com/knative/docs/pull/4189

so we can close this PR

**Comment by JeffNeff** [comment]
@snneji actually had a PR open for the same fix xd. I think I still like my fix more tho because it prevents us from having to do this again on the next release.. Im also fine with closing this tho :) @sebgoa  thoughts?

**Comment by snneji** [comment]
Hi @JeffNeff, in the end we decided that it's easier for readers to see the K8s version number explicitly in the docs, even though we will need to keep this up to date. I'm going to close this PR since #4189 has been merged.

**Pull request closed without merging**